src/main/java/core/basesyntax/shape/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class Square extends Shape {
+    private double side;
+
+    public Square(Color color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return this.side * this.side;
```

Comment:
You can remove the 'this' keyword when accessing instance variables, as it's not necessary in this case.